scripts/config/azure.yaml

```diff
@@ -6,6 +6,7 @@ demikernel:
   local_link_addr: "ff:ff:ff:ff:ff:ff"
 raw_socket:
   interface_name: "abcde"
+  interface_index: 0
```

Comment:
should we annotate this with linux_interface_name and xdp_interface_index?

Reply:
Good idea! Will change this and the consuming code.